python/dgl/graphbolt/impl/fused_csc_sampling_graph.py

```diff
@@ -934,6 +934,61 @@ def sample_negative_edges_uniform(
             ),
         )
 
+    def sample_negative_seeds_uniform(
```

Comment:
We'd still call it sample_negative_edges_uniform, we are start from sample_negative_edges_uniform_2, and rename later.